pkg/app/server/grpcapi/web_api.go

```diff
@@ -1434,7 +1434,7 @@ func (a *WebAPI) ListAPIKeys(ctx context.Context, req *webservice.ListAPIKeysReq
 
 // GetInsightData returns the accumulated insight data.
 func (a *WebAPI) GetInsightData(ctx context.Context, req *webservice.GetInsightDataRequest) (*webservice.GetInsightDataResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+	return nil, status.Error(codes.Unimplemented, "GetInsightData API is not yet implemented.")
```

Comment:
How about `Not yet implemented` or `Not available`?

Reply:
Lets make it `Not available` 😄

Reply:
Since this message is directly displayed in the UI, I was wondering if it would be better to clarify to the user what is not yet implemented.
What do you think?

Reply:
same as your reason, but I think end-users should not be annoyed by which endpoint/API is not yet implemented, so generally mentioning that the feature/UI is not available is enough to me. We - as the dev - should know about what is not yet implemented (by checking the network debugging tool if it's an HTTP request, but in this case gRPC only returns 200 even in case it's an error 😓 ).
In short, I think users don't care which API is failed, only us care about that and we should not annoy users by making those kind of message 😄

Reply:
I understand! then I would like to update it with `Not available` 👍

Reply:
now it's done!
![image](https://user-images.githubusercontent.com/26561120/170921431-dcc6b2e9-1546-49e6-ab2f-578dcdbc3bba.png)